user/tutorial.md

```diff
@@ -41,6 +41,9 @@ To start using Travis CI, make sure you have:
    The defaults for Ruby projects are `bundle install` to [install dependencies](/user/job-lifecycle/#customizing-the-installation-phase),
    and `rake` to build the project.
 
+    * NB: If you want to use a YAML header line, make it `%YAML 1.1`.
```

Comment:
This caution about the YAML version probably does not belong here. The tutorial is meant to be followed step by step, and the YAML document above is YAML 1.1-compliant.

Reply:
Of course you're technically right. It's a matter of how to deal with real users not obeying your expectation, scrolling down until they find something that seems to fit their search. Would you prefer some of their time wasted as the rightful punishment, or do you make your docs to help them get their job done?

Reply:
By this reasoning, we would be including the complete reference of the `.travis.yml` structure.

This statement belongs in our documents, but not in the tutorial.

Reply:
Ok, thanks for the explanation then. It's a great project nonetheless!

Reply:
Actually, I just checked again to see how bad a reader I must have been. But in current master, even when reading from the top, I can't find information about what file format the `.travis.yml` is meant to be. The `.yml` obviously isn't enough of a hint if only certain versions of YAML are supported. Let's at least have a link to the reference there.